extensions/dicom-sr/src/OHIFCornerstoneSRViewport.js

```diff
@@ -181,28 +187,36 @@ function OHIFCornerstoneSRViewport({
         cornerstone.updateImage(element);
       }
     });
-  };
+  });
 
-  useEffect(() => {
-    if (element !== null) {
-      setTrackingUniqueIdentifiersForElement(element);
-    }
-  }, [
-    dataSource,
-    displaySet,
-    displaySet.StudyInstanceUID,
-    displaySet.displaySetInstanceUID,
-  ]);
+  useEffect(
+    () => {
+      if (element !== null) {
+        setTrackingUniqueIdentifiersForElement(element);
+      }
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [
+      dataSource,
+      displaySet,
+      displaySet.StudyInstanceUID,
```

Comment:
We are already tracking the `displaySet` in this useEffect, so these last two lines aren't needed?

Reply:
fixed.